README.md

```diff
@@ -3,14 +3,47 @@
 This repository contains assets that are binary files or too large to be checked in directly. Packages are uploaded to the default transport channel's feed.
 
 ## Workflow
+
 1. Submit a PR with the new assets.
 2. After the PR is merged, an internal build publishes assets to the Build Assets Registry and an auto PR will be opened in the subscribing repositories.
 
+- Optional step: Local testing.
+
 ## Example
-We are working on adding a new unit test for the GZip feature from the `System.IO.Compression` namespace, and the test depends on a file called `example.gz`.
 
-#### 1. Submit a PR with the new assets.
-Save `example.gz` inside `runtime-assets\src\System.IO.Compression.TestData\GZipTestData`.
+Let's assume we are adding a new unit test for the GZip feature from the `System.IO.Compression` namespace, and the test depends on a file called `example.gz`.
+
+### 1. Submit a PR to runtime-assets with the new files.
 
 ### 2. Post-merge
-After the PR is merged an internal build will publish the packages to an internal feed and update the Build Assets Registry. Subscribers to this repository will receive an auto PR update with the updated package version.
\ No newline at end of file
+
+After the PR is merged an internal build will publish the packages to a public feed and update the Build Assets Registry. Subscribers to this repository will receive an auto PR update with the updated package version.
+### Optional step: local testing
+
+Testing your assets in your local machine is an optional step you can perform before submitting your PR.
+Local testing is usually not necessary due to the nature of this repository, which mostly only deals with static files.
+Feel free to skip this section.
+
+a) Save `example.gz` inside `runtime-assets\src\System.IO.Compression.TestData\GZipTestData`.
+
+b) Run `build.cmd --pack` from the `runtime-assets\` root folder.
+
+c) Verify that your package was generated with the new test data.
+  I.  Go to `runtime-assets\artifacts\packages\Debug\NonShipping`.
+  II.  Temporarily rename `System.IO.Compression.TestData.7.0.0-dev.nupkg` to the extension `*.nupkg.zip`.
+  III.  Open it with your preferred Zip app.
+  -  Navigate to `contentFiles\any\any\GZipTestData` and verify that the new `example.gz` file can be found there.
+  IV.  Close the Zip window, and revert the name to `*.nupkg`.
+  V.  Copy the `System.IO.Compression.TestData.7.0.0-dev.nupkg` to a temporary folder, for example, `D:\tmp\`.
+
+d)  Go to your `runtime` folder.
+
+e)  Edit `Nuget.config` located in the root folder, and inside the `<packageSources>` section, add a new key that points to the temporary folder as a source: `<add key="tmp" value="D:\tmp" />`.
```

Comment:
based on my suggestion above you would need to update this so that the example points to the `runtime-assets\artifacts\packages\Debug\NonShipping` location.

Reply:
@safern Addressed. Looks good now?